### PR TITLE
add tiered locality sceduling docs and set the default tiered locality configuration

### PR DESCRIPTION
**Pull request by xliuqq**
### Ⅰ. Describe what this PR does
Add tiered locality sceduling docs and set the default tiered locality configuration.

### Ⅱ. Does this pull request fix one issue?

fixes NONE

### Ⅴ. Special notes for reviews

### CODE REVIEW

File: charts/fluid/fluid/values.yaml [hunk 1]
```diff
@@ -152,12 +152,6 @@ webhook:
       # fluid existed node affinity, the name can not be modified.
       - name: fluid.io/node
         weight: 100
-      # runtime worker's rack label name, can be changed according to k8s environment.
-      - name: topology.kubernetes.io/rack
-        weight: 50
-      # runtime worker's zone label name, can be changed according to k8s environment.
-      - name: topology.kubernetes.io/zone
```
Comment (cheyang) [inline charts/fluid/fluid/values.yaml:159]: Can you change the default configuration as

```yaml
# runtime worker's rack label name, can be changed according to k8s environment.
      - name: topology.kubernetes.io/zone
        weight: 50
      # runtime worker's zone label name, can be changed according to k8s environment.
      - name: topology.kubernetes.io/region
         weight: 20
     ```

**Review by cheyang — CHANGES_REQUESTED** [review]

File: docs/en/operation/tiered_locality_schedule.md [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+Install Fluid following the document [Installation](../userguide/install.md). After installation, a configmap
+named `tiered-locality-config` storing above configuration will exist in Fluid namespace(default `fluid-system`).
+
+2) Modify tiered locality configuration in the existing Fluid cluster
+
+Modify tiered location configuration (content see point 1) in the configMap named 'tiered local configuration' 
+in the Fluid namespace (default `fluid-system`), the latest configuration will be read for Pod scheduling during the next webhook mutation.
+
+## 2. Configure the tiered locality information for the Runtime
+可以通过 Dataset 的 nodeAffinity 或者 Runtime 的 NodeSelector 字段配置分层位置信息。
```
Comment (cheyang) [inline docs/en/operation/tiered_locality_schedule.md:59]: Please use English to describe. thanks.

File: charts/fluid/fluid/values.yaml (hunk 1 above)
Comment (xliuqq) [reply charts/fluid/fluid/values.yaml:159]: fixed.

File: docs/en/operation/tiered_locality_schedule.md (hunk 2 above)
Comment (xliuqq) [reply docs/en/operation/tiered_locality_schedule.md]: fixed.

**Review by cheyang — APPROVED** [review]
/lgtm
/approve

**Pull request merged**